rogueutil.h

```diff
@@ -876,7 +877,16 @@ getUsername(void)
                 return ret;
         return NULL;
 #else /* _WIN32 */
-        return getlogin();
+#ifdef __linux__
+        struct passwd *pw = getpwuid(getuid());
+				if (pw) {
+					return pw->pw_name;
+				} else {
+					return NULL;
+				}
+#else
+				return getlogin():
+#endif
```

Comment:
Something happened to the formatting here. Are you using tabs, as recommended by the style guidelines?

Reply:
Four tabs as the win32 section of the function? Right? I didn't see that the editor is using spaces instead of tabs

Reply:
Single tab, like the win32 section.

Reply:
It should already be corrected, please check it. Clion broke the indentation somehow when opening and saving the file.

Reply:
Looks resolved in the recent commit! :)